### PR TITLE
Add Standard Notes

### LINKED ISSUE #6670: Unsure if site qualifies to be added
I was looking into submitting a pr for standardnotes.com, but after reading the Site Criteria I am not too sure if it qualifies to be added. I think it qualifies under the second criteria, Data. Can I get another opinion on this.

### PR DESCRIPTION
**Pull request by aarontorres0**

### CODE REVIEW

File: entries/s/standardnotes.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "Standard Notes": {
+        "domain": "standardnotes.com",
+        "tfa": [
+            "totp"
+        ],
+        "documentation": "https://standardnotes.com/help/25/how-do-i-enable-two-factor-authentication-for-my-account",
```
Comment (phallobst) [inline entries/s/standardnotes.com.json]:
```suggestion
        "documentation": "https://standardnotes.com/help/25",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/s/standardnotes.com.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="258.40 258.40 507.20 507.20">
+   <defs id="defs16"></defs>
+   <g id="Icon">
+      <g id="Logo_recreated_by_krisu_mod_by_miagit87" style="fill:#086dd6;fill-opacity:1">
+               <path class="st1" d="M726.6 297.4v429.2H297.4V297.4h429.2m39-39H258.4v507.2h507.2V258.4z" id="path6" style="fill:#086dd6;fill-opacity:1"></path>
+               <path class="st1" d="M523.8 404.6h-23.7c-43.9 0-79.5 35.6-79.5 79.5v135.5h37.8V484.8c0-25.7 20.8-46.5 46.5-46.5h13.9c25.7 0 46.5 20.8 46.5 46.5v134.6h37.8V484c.2-43.9-35.4-79.4-79.3-79.4z" id="path8" style="fill:#086dd6;fill-opacity:1"></path>
+      </g>
+   </g>
+</svg>
```
Comment (Carlgo11) [inline img/s/standardnotes.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="258.4 258.4 507.2 507.2"><path fill="#086dd6" d="M726.6 297.4v429.2H297.4V297.4h429.2m39-39H258.4v507.2h507.2V258.4zM523.8 404.6h-23.7c-43.9 0-79.5 35.6-79.5 79.5v135.5h37.8V484.8c0-25.7 20.8-46.5 46.5-46.5h13.9c25.7 0 46.5 20.8 46.5 46.5v134.6h37.8V484c.2-43.9-35.4-79.4-79.3-79.4z"/></svg>
```

File: entries/s/standardnotes.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "Standard Notes": {
+        "domain": "standardnotes.com",
+        "tfa": [
+            "totp"
+        ],
+        "documentation": "https://standardnotes.com/help/25",
```
Comment (Carlgo11) [inline entries/s/standardnotes.com.json:7]:
```suggestion
        "documentation": "https://standardnotes.com/help/25",
        "recovery": "https://standardnotes.com/help/22",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/s/standardnotes.com.json (hunk 3 above)
Comment (aarontorres0) [reply entries/s/standardnotes.com.json:7]: It says these changes are invalid because no change was made to the code

Comment (aarontorres0) [reply entries/s/standardnotes.com.json:7]: I just went ahead and added this myself

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**